packages/apps/escrow-dashboard/src/constants/index.ts

```diff
@@ -58,6 +81,7 @@ export const SUPPORTED_CHAIN_IDS = [
   ChainId.MOONBASE_ALPHA,
   ChainId.AVALANCHE_TESTNET,
   ChainId.AVALANCHE,
+  ChainId.SKALE,
```

Comment:
please remove it from here until we have a subgraph maintainer

Reply:
done !